virtcontainers/qemu_amd64.go

```diff
@@ -93,7 +93,8 @@ func newQemuArch(config HypervisorConfig) qemuArch {
 		factory = true
 	}
 
-	var qemuMachines = supportedQemuMachines
+	qemuMachines := make([]govmmQemu.Machine, len(supportedQemuMachines))
```

Comment:
This change only fix the issue for amd64 arch for qemu hypervisor, but there are other archs which still had this issue.  Is it better to fix it by checking whether 	appending the qemuNvdimmOption  needed 
 in https://github.com/kata-containers/runtime/blob/cd63aacc9eaf6b59d32f900fe875d949a55e1b4d/virtcontainers/qemu_arch_base.go#L706, and just append the option when it didn't exist in q.supportedQemuMachines[i].Options?

Reply:
@lifupan @jodh-intel thanks very much
>there are other archs which still had this issue

I agree with you and I have add the change for other archs. 

> just append the option when it didn't exist
Functionally, it's no problem now, but if not copy, the content of supportedQemuMachines is changed，it could be the place where things go wrong in the future.